sequencer/src/api/endpoints.rs

```diff
@@ -46,6 +46,42 @@ pub struct NamespaceProofQueryData {
     pub transactions: Vec<Transaction>,
 }
 
+pub(super) fn get_balance<State, Ver>() -> Result<Api<State, merklized_state::Error, Ver>>
```

Comment:
Add at least a TODO comment that mentions that this code is currently not covered by any tests.

Reply:
there is a test, as part of slow_test_merklized_state_api test